scripts/pylib/twister/twisterlib.py

```diff
@@ -3074,6 +3108,11 @@ def apply_filters(self, **kwargs):
                     # Discard silently
                     continue
 
+                test_configuration = ".".join([instance.platform.name,
+                                               instance.testcase.id])
+                if test_configuration in self.quarantine:
+                    discards[instance] = discards.get(instance,
+                                                      "Quarantine")
```

Comment:
just wonder whether it is possible to add the github issue link into report, so that we can tarck the latest status

Reply:
It is a good idea to be able to easily track related bugs. The simplest, for now, is to add comments into quarantine.yaml with the related links and whatever information the user wants. Is the link useful in the report? We can discuss this during the testing wg meeting.